packages/@n8n/task-runner/src/js-task-runner/js-task-runner.ts

```diff
@@ -468,6 +505,8 @@ export class JsTaskRunner extends TaskRunner {
 		dataProxy: IWorkflowDataProxyData,
 		additionalProperties: Record<string, unknown> = {},
 	): Context {
+		this.preventPrototypePollution();
```

Comment:
We don't need to run this for every execution. It's enough to do it once during startup

Reply:
[d9a5396](https://github.com/n8n-io/n8n/pull/12588/commits/d9a5396138d8b906c1745de4e7de31b965760e05)